src/views/v2/day/top-bar/nav/next-disabled.php

```diff
@@ -20,5 +20,6 @@ class="tribe-common-c-btn-icon tribe-common-c-btn-icon--caret-right tribe-events
 		title="<?php esc_attr_e( 'Next day', 'the-events-calendar' ); ?>"
 		disabled
 	>
+		<?php $this->template( 'components/icons/caret-right', [ 'classes' => [ 'tribe-common-c-btn-icon-svg', 'tribe-common-c-btn-icon-svg--caret-right', 'tribe-events-c-top-bar__nav-link-icon-svg' ] ] ); ?>
```

Comment:
In sticking with BEM (from common) and since the button itself has a modifier, we don't need an extra modifier here. This applies to all the buttons.

```suggestion
		<?php $this->template( 'components/icons/caret-right', [ 'classes' => [ 'tribe-common-c-btn-icon__icon-svg', 'tribe-events-c-top-bar__nav-link-icon-svg' ] ] ); ?>
```